src/pkg/cli/subscribe.go

```diff
@@ -38,14 +37,6 @@ func Subscribe(ctx context.Context, client client.Client, services []string) (<-
 		defer serverStream.Close()
 		defer close(statusChan)
 		for {
-			// handle cancel from caller; TODO: do we need this? Receive() should return false on context done
-			select {
-			case <-ctx.Done():
-				term.Debug("Context Done - exiting Subscribe goroutine")
```

Comment:
avoid printing in go-routine since it will mess up other prints in main program

Reply:
This is more of a failure of our logging package than requirement, but I agree we should keep where logging happens, and try to avoid spread logging into all layers of the dependencies.